app/furniture/marketplace/locales/en.yml

```diff
@@ -55,9 +55,10 @@ en:
       create:
         success: "Added Product '%{name}'"
       edit:
-        link_to: "Edit Product '%{name}'"
+        link_to: "Change"
+        link_to: "Change Product '%{name}'"
```

Comment:
IMHO `Edit` is clearer than `change` especially for modifying the details of a product. `Change` IMHO makes it sound like the product is changing to a different product.

Reply:
But I'm also okay w/ this terminology change (pun intended 😉 ) too .... so 🤷‍♀️

Reply:
Yeah, I think I like "Edit" a bit better as well. Feels more standard, less surprising/confusing.

Reply:
I am down for keeping it as `Edit` my main goal here was consolidating it into a top-level `edit.link_to` which always just says "Edit" which we can use when we don't want to define a custom `<resource>.edit.link_to`